packages/web/src/pages/sign-up-page/pages/SignUpPage.tsx

```diff
@@ -1,20 +1,45 @@
-import { useCallback } from 'react'
+import { useCallback, useEffect } from 'react'
 
-import { Button, ButtonType, Text } from '@audius/harmony'
-import { Form, Formik } from 'formik'
+import { Status } from '@audius/common'
+import { useEmailInUse } from '@audius/common/src/api/signUp'
+import {
+  Button,
+  ButtonType,
+  Divider,
+  Flex,
+  IconArrowRight,
+  Text,
+  TextInput
+} from '@audius/harmony'
+import { useFormik } from 'formik'
+import { useDispatch } from 'react-redux'
+import { z } from 'zod'
+import { toFormikValidationSchema } from 'zod-formik-adapter'
 
-import { TextField } from 'components/form-fields'
+import audiusLogoColored from 'assets/img/audiusLogoColored.png'
 import { Link } from 'components/link'
-import { SIGN_IN_PAGE } from 'utils/route'
+
+import { setValueField } from '../../../common/store/pages/signon/actions'
+import PreloadImage from '../../../components/preload-image/PreloadImage'
+import { useNavigateToPage } from '../../../hooks/useNavigateToPage'
+import { EMAIL_REGEX } from '../../../utils/email'
+import { SIGN_IN_PAGE } from '../../../utils/route'
+import { SocialButton } from '../components/SocialButton'
 
 import { CreatePasswordState } from './CreatePasswordPage'
+import styles from './SignUpPage.module.css'
 
 const messages = {
-  header: 'Sign Up For Audius',
+  title: 'Sign Up For Audius',
   emailLabel: 'Email',
   signUp: 'Sign Up Free',
   haveAccount: 'Already have an account?',
-  signIn: 'Sign In'
+  signIn: 'Sign In',
+  subHeader:
+    'Join the revolution in music streaming! Discover, connect, and create on Audius.',
+  dividerText: 'Or, get started with one of your socials',
+  invalidEmail: 'Please enter a valid email.',
+  unknownError: 'Unknown error occurred.'
```

Comment:
Note: need to check with design if they have a preferred messaging here. I'm also assuming general network issues would be the only way to trigger this message?